src/client/mapblock_mesh.cpp

```diff
@@ -969,9 +972,13 @@ video::SColor encode_light(u16 light, u8 emissive_light)
 		r = day * 255 / sum;
 	else
 		r = 0;
+	if (r < 0xF0 && r&0b1100)
+		r += 0x10;
 	// Average light:
-	float b = (day + night) / 2;
-	return video::SColor(r, b, b, b);
+	u32 b = sum / 2;
+	if (b < 0xF0 && b&0b1100)
+		b += 0x10;
+	return video::SColor((r&0xF0)|(b>>4), 0, 0, 0);
```

Comment:
And this should have 0xFF's instead of those 0's.

Reply:
yeah I like the different lighting colour

Reply:
🤦